### PR TITLE
feat(sortFacetValuesBy): add sortFacetValuesBy query parameter

### PR DESCRIPTION
**Pull request by spinach**

### CODE REVIEW

File: Sources/AlgoliaSearch-Client/Query.swift [hunk 1]
```diff
@@ -805,6 +805,28 @@ open class Query : AbstractQuery {
...
+        /// Facet values are sorted by decreasing count, the count being the number of records containing this facet value in the results of the query (default behavior).
+        case count = "count"
+        /// Facet values are sorted by increasing alphabetical order.
+        case alpha = "alpha"
+    }
+    
+    /// Controls how the facet values are sorted within each faceted attribute.
+    public var sortFacetValuesBy: SortFacetValuesBy? {
+        get {
+            if let value = self["sortFacetValuesBy"] {
```
Comment (robertmogos) [inline Sources/AlgoliaSearch-Client/Query.swift]: guard let maybe? both way work, I guess

File: Sources/AlgoliaSearch-Client/Query.swift [hunk 2]
```diff
@@ -1106,4 +1128,10 @@ open class Query : AbstractQuery {
         get { return AbstractQuery.toNumber(self.percentileComputation) }
         set { self.percentileComputation = newValue?.boolValue }
     }
+    
+    @objc(sortFacetValuesBy)
+    public var z_objc_sortFacetValuesBy: String? {
+        get { return sortFacetValuesBy?.rawValue }
+        set { sortFacetValuesBy = newValue == nil ? nil : SortFacetValuesBy(rawValue: newValue!) }
```
Comment (robertmogos) [inline Sources/AlgoliaSearch-Client/Query.swift]: I would make it explicit with an if or guard. Even though is "longer" it's safer. Here, a miss matched equal could "break" the code

File: Sources/AlgoliaSearch-Client/Query.swift (hunk 1 above)
Comment (spinach) [reply Sources/AlgoliaSearch-Client/Query.swift]: I like guard, more compact and less indentation

File: Sources/AlgoliaSearch-Client/Query.swift (hunk 2 above)
Comment (spinach) [reply Sources/AlgoliaSearch-Client/Query.swift]: discussed 1:1

**Pull request merged**